config.go

```diff
@@ -55,7 +56,11 @@ func NewConfig(alias, keyStorePath, password string, address *Address, databaseP
 var logger *logrus.Logger
 
 func init() {
+	maxUint256 = new(big.Int)
```

Comment:
We leave it like it is, but just as a remark, there can be several `init()` functions in several files in the same package. So it would have been cleaner to put this into its own `init()` in `client.go`.

Reply:
I was not sure about that but also thought about it.